memegen/codelab/memegen_codelab.md

```diff
@@ -578,7 +578,21 @@ Follow the steps:
                     \image_filepath TEXT)")
    ```
 
-   Make sure to add *sqlite-simple* as a Cabal dependency.
+   Make sure to
+  
+      * add *sqlite-simple* as a Cabal dependency
+      * expose the ```Memegen.Db``` module:
```

Comment:
Could you fit this on a single line?
"Expose Memegen.Db module and add *sqlite-simple* as a Cabal dependency:"